### PR TITLE
Create Set-TeamTargetingHierarchy.md

### PR DESCRIPTION
**Pull request by ayrtonsoftware**

### CODE REVIEW

File: teams/teams-ps/teams/Set-TeamTargetingHierarchy.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+## SYNTAX
+
+### Get (Default)
+```
+Set-TeamTargetingHierarchyStatus [-FilePath <String>]
+```
+
+## DESCRIPTION
+
+A sample CSV file uses the following format:
```
Comment (tseward) [inline teams/teams-ps/teams/Set-TeamTargetingHierarchy.md:25]: Keep in mind that markdown and HTML tags will not render properly in PowerShell.

File: teams/teams-ps/teams/Set-TeamTargetingHierarchy.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+> - Apogee <br>  - New Jersey <br>    - Basking Ridge <br>    - Moutain Lakes
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Set-TeamTargetingHierarchy -FilePath d:\hier.csv
+
+Key       Value
```
Comment (tseward) [inline teams/teams-ps/teams/Set-TeamTargetingHierarchy.md:39]: Output from a cmdlet should be encased in ```output:

\```powershell
\> Cmdlet
\```

\```output
Result
\```

**Review by tseward — CHANGES_REQUESTED** [review]

File: teams/teams-ps/teams/Set-TeamTargetingHierarchy.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+---
+external help file: Microsoft.Open.Teams.CommonLibrary.dll-Help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/powershell/module/teams/connect-microsoftteams
+schema: 2.0.0
+author: mbergamo
```
Comment (get-itips) [inline teams/teams-ps/teams/Set-TeamTargetingHierarchy.md:6]: author metadata should be a github username

**Review by get-itips — CHANGES_REQUESTED** [review]
small suggestions and also see Trevor's

**Review by dariomws — APPROVED** [review]
I'm ok after suggestions.

**Comment by yogkumgit** [comment]
@ayrtonsoftware

**Comment by yogkumgit** [comment]
@ayrtonsoftware Please update the PR with suggestions.

**Pull request merged**